.github/workflows/sync.yml

```diff
@@ -0,0 +1,194 @@
+name: Sync
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+env:
+  BOT_USERNAME: "exercism-bot"
+  GIT_USERNAME: "Exercism Bot"
+  GIT_EMAIL: "66069679+exercism-bot@users.noreply.github.com"
+
+jobs:
+  fetch-tracks:
+    name: Fetch & fork tracks
+    runs-on: ubuntu-20.04
+
+    outputs:
+      tracks: ${{ steps.fetch-tracks.outputs.result }}
+
+    steps:
+      - name: Fetch list of tracks
+        id: fetch-tracks
+        uses: actions/github-script@47f7cf65b5ced0830a325f705cad64f2f58dddf7
+        with:
+          script: |
+            const trackRepos = await github.paginate(github.search.repos, {
+              q: 'org:exercism+topic:exercism-track'
```

Comment:
I'm not saying we should, but have we considered if we want to apply this to private or archived repositories? @iHiD

Reply:
I think we should stick to public.

Reply:
Then we must add `is:public` to the query